### PR TITLE
RFC for 2023H2 Roadmap

### PR DESCRIPTION
**Pull request by hone**
[Readable](https://github.com/buildpacks/rfcs/blob/9ff4797e8091288afa916a7fee2eb1332d398819/text/0000-2023H2-roadmap.md)

### CODE REVIEW

File: text/0000-2023H2-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+The RFC has been merged and the buildpacks community GitHub org have been created.
+
+#### Pack Test Cleaning/Optimizations
+* Owner: @dfreilich
+* Status: Defer
+* Links: [Pack Pull Request](https://github.com/buildpacks/pack/pull/1498)
+
+This item has been deferred for now, but we will work with anyone who wants to push this forward.
+
```
[the remark this answers is not in the document]
Comment (hone) [reply text/0000-2023H2-roadmap.md:76]: Added in [6d37bbc](https://github.com/buildpacks/rfcs/pull/286/commits/6d37bbcde404abd83fcd89f9b91e747a90d6c76f)

**Comment by hone** [comment]
With @jjbustamante's addition, I'd like to move this to voting to close July 6.

**Review by jjbustamante — APPROVED** [review]
LGTM

**Review by natalieparellano — APPROVED** [review]

File: text/0000-2023H2-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Currently, Buildpack Authors have little to no tools around visibility with their buildpacks as they run on a platform, including `pack`. In some of the Heroku v2 buildpacks, they implemented logging that could be handed off to the platform by running `bin/report`. This work stream is about standardizing output for both successful AND failed builds that Buildpack Authors can use to instrument their buildpack.
+
+#### Private Registry Mirrors
+* Owner: @jabrown85
+* Links: [RFC](https://github.com/buildpacks/rfcs/pull/285)
+
+A platform operator can configure registry mirrors that lifecycle could use without needing for the manifest to have to point to it. This will allow a platform to reduce the risk of service operations from external registry sources and reduce public network bandwidth usage. The resulting image when taken off platform will also function without needing access to the registry mirror.
+
+#### kpack Donation
+* Owner: @jjbustamante (Team Lead sponsor @hone)
```
Comment (samj1912) [inline text/0000-2023H2-roadmap.md:98]: @hone would you be okay with this being me? 
```suggestion
* Owner: @jjbustamante (Team Lead sponsor samj1912)
```

**Review by samj1912 — APPROVED** [review]
Minor comment but lgtm

**Review by jkutner — APPROVED** [review]

File: text/0000-2023H2-roadmap.md (hunk 2 above)
Comment (hone) [reply text/0000-2023H2-roadmap.md]: @samj1912 :+1:

**Pull request merged**